README.md

```diff
@@ -30,15 +29,6 @@ node.
 cargo run --release -- --dev --tmp --mining-key <your mining key>
 ```
 
-### Build
```

Comment:
What's the reason for removing this?

Reply:
b/c further up in this file we have 
```
First, you must complete the [basic environment setup](/docs/dev-guide/src/getting-started/building.md#build-prerequisites).
```

and the devel guide shows more info around building and what rustc version to use.

Also b/c `cargo run --release` mentioned above will do the build for you.